src/view/draggable/connected-draggable.js

```diff
@@ -156,22 +159,33 @@ export const makeSelector = () => {
           return defaultMapProps;
         }
 
-        if (pending.last.current.id !== id) {
+        if (pending.result.draggableId !== id) {
+          // This flag is a matter of degree.
```

Comment:
A little rough edge that try to balance correctness with an a constantly interactive application